input/_partials/_footer.cshtml

```diff
@@ -40,3 +44,4 @@
     </div>
   </div>
 </footer>
+<script src="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"></script>
```

Comment:
I thought we already had the font-awesome content included. Do we need this again here?

Reply:
Didn't realized at all... yeah, header has it, sorry! Just updated my PR with that correction